src/plugins/music/music-regions-plugin.ts

```diff
@@ -10,9 +10,9 @@ export function playSongForRegion(player: Player): void {
     const songId: number = musicRegionMap.get(regionId);
     logger.info('Song ID for this region: ' + songId);
     logger.info('Size: ' + regionId);
-    player.sendMessage('Song playing: ' + getByValue(songs, songId + 1));
+    player.sendMessage('Song playing: ' + getByValue(songs, songId));
```

Comment:
This should be removed, i'll leave it for now.

Reply:
Oh, whoops. That's a good point. Should've removed that beforehand.

Reply:
Yeah probably, remove them and ill merge :)